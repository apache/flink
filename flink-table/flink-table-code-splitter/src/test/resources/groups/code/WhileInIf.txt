{
    a[0] += b[1];
    b[1] += a[1];
    if (a.length < 100) {
      while (counter > 0) {
          myFun_whileBody0_0(a, b);
          if (a[0] > 0) {
              myFun_whileBody0_0_ifBody0(a, b);
          } else {
             myFun_whileBody0_0_ifBody1(a, b);
          }
          counter--;
      }

      a[2] += b[2];
      b[3] += a[3];
    } else {
      while (counter < 100) {
          b[4] = b[4]++;
          counter++;
      }
    }

    a[5] += b[5];
    b[6] += a[6];
}
