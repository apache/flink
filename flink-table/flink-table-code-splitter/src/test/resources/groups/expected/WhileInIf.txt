{
  myFun_rewriteGroup8(a, b);

  myFun_rewriteGroup9(a, b);
}
